project-base/UPGRADE.md

```diff
@@ -1,6 +1,14 @@
 # Upgrading
 
 ## From 7.0.0-alpha1 to Unreleased
+- check changes in the `docker-compose.yml` template you used, there were a couple of important changes you need to replicate
+    - easiest way is to overwrite your `docker-compose.yml` with by the appropriate template
+- on *nix systems, fill your UID and GID (you can run `id -u` and `id -g` to obtain them) into Docker build arguments `www_data_uid` and `www_data_gid` and rebuild your image via `docker-compose up --build`

```

Comment:
I am missing the `-d` argument here, it is little bit confusing and I think it should be `docker-compose up -d --build`